index.js

```diff
@@ -47,6 +47,12 @@ if (keepLatest === 0) {
   console.error("🌶  given `keep_latest` is 0, this will wipe out all releases");
 }
 
+const shouldDeleteTags = process.env.INPUT_DELETE_TAGS === "true";
```

Comment:
why the term "INPUT" as well?

Reply:
It is internal to github action. option name will be `delete_tags` (lowercase)